package.json

```diff
@@ -78,7 +78,8 @@
     ]
   },
   "dependencies": {
-    "@typescript-eslint/experimental-utils": "^4.0.1"
+    "@typescript-eslint/experimental-utils": "^4.0.1",
+    "is-ci": "^3.0.0"
```

Comment:
dev dep

Reply:
(`pinst` deals with it so it's not actually published as a postinstall)

Reply:
(it'd be nice if `yarn` v2 supported `prepare` 😞)